src/components/map/Mapbox.vue

```diff
@@ -9,23 +9,26 @@
     @load="onMapLoaded"
   >
     <MapMarker
-      v-for="(m, index) in events"
+      v-for="(event, index) in events"
       :key="`marker-${index}`"
-      :name="m.pub.name"
-      :coords="m.priv.coordinates"
-      :img-src="m.mediaLink.cover"
+      :name="event.pub.name"
+      :coords="event.priv.coordinates"
+      :img-src="event.mediaLink.cover"
+      @navigate="navigateToEvent(event)"
```

Comment:
shouldn't pass an `object` here as the params, because it'll come out as something like `%5Bobject%20Object%5D678`.  suggest doing

> navigateToEvent(`${event.pub.name}${event.id}`)


or something similar

Reply:
ok lets do it like that i didnt know this works

Reply:
actually if u know how to do it can u just commit cuz im actually not sure how to do this

Reply:
should we or should we not use router push?

Reply:
Ooohh sure, i'll commit on the `event-details-page`.

Using router push is ok but basically like we can't pass an object as a string to url because an object that isn't stringified will just appear as `%__Object__%` or something. So I'm gonna change it to just pass the event id to the url - there's a related comment where when we start integrating the API, we'll need to get this url param on loading the 'eventpage' and use it to make a request to get the event data

Reply:
Ok sorry actually I severely misread the code - I think the problem needs to be addressed only once we integrate API calls into front-end. I just saw a weird url and made assumptions about how it was written